prow/cmd/phaino/main.go

```diff
@@ -116,58 +107,42 @@ func readHTTP(ctx context.Context, url string) (*prowapi.ProwJob, error) {
 	if resp.StatusCode < 200 || resp.StatusCode > 299 {
 		return nil, fmt.Errorf("bad status: %d", resp.StatusCode)
 	}
-	return readPJ(ctx, resp.Body)
+	return readPJ(resp.Body)
 }
 
-func readPJs(ctx context.Context, jobs []string) (<-chan prowapi.ProwJob, <-chan error) {
+func readPJs(jobs []string) (<-chan prowapi.ProwJob, <-chan error) {
 	ch := make(chan prowapi.ProwJob)
 	errch := make(chan error)
 	go func() {
 		defer close(ch)
 		defer close(errch)
 		if len(jobs) == 0 {
 			logrus.Info("Converting stdin prowjob...")
-			pj, err := readPJ(ctx, os.Stdin)
+			pj, err := readPJ(os.Stdin)
 			if err != nil {
-				select {
-				case errch <- err:
-				case <-ctx.Done():
-				}
-				return
-			}
-			select {
-			case ch <- *pj:
-			case <-ctx.Done():
+				errch <- err
```

Comment:
I think there needs to be a return after this.

Reply:
Done